subcommand/lifecycle-sidecar/command.go

```diff
@@ -98,22 +111,24 @@ func (c *Command) Run(args []string) int {
 	//
 	// The loop will only exit when the Pod is shut down and we receive a SIGINT.
 	for {
-		cmd := exec.Command(c.flagConsulBinary, c.consulCommand...)
+		start := time.Now()
+		cmd := exec.CommandContext(ctx, c.flagConsulBinary, c.consulCommand...)
 
 		// Run the command and record the stdout and stderr output
 		output, err := cmd.CombinedOutput()
+		// Currently this command is taking >7s, need to investigate
+		logger.Debug("time to run resync:", "time", time.Since(start))
```

Comment:
I know this was just for debugging but just making a note that we should clean it up.